### ISSUE TITLE
Incorrect TreeView scrollviewer behavior with Windows UI scaling on

### ISSUE THREAD

**Issue by Al-Dyachkov** [AvaloniaCommunity/Material.Avalonia#108]
Following code produces visible scrollbars with ui scaling on and no visible scrollbars with ui scaling off
Vanilla Avalonia behaves as expected, no visible scrollbars
Avalonia 10.7, Material.Avalonia 2.3.0

Video to see differences : https://www.youtube.com/watch?v=ZAbxUv-H2bg
Right part is a screen with 125% scaling, left part is 100% scaling

Code to reproduce : just create blank avalonia project with following window contents
```xaml
    <Border>
          <TreeView MinWidth="300" MinHeight="300" VerticalAlignment="Center" HorizontalAlignment="Center"/>
    </Border>
```

**Comment by appleneko2001** [comment]
Try this one! I just pushed an temporarily fixes for this. The reason of this bug happens because there was an style setter `<Setter Property="Padding" Value="1" />` in TreeView.xaml. So I replaced the value with 0, and this bug disappears. 

https://github.com/AvaloniaCommunity/Material.Avalonia/actions/runs/1202807356

https://user-images.githubusercontent.com/38249047/132122899-512045e6-75c2-4659-8cfc-8c3f7b0464ef.mp4
